src/index.ts

```diff
@@ -293,7 +299,11 @@ async function statusOk(res: Response, logger: Logger): Promise<boolean> {
     return String(res.status)[0] === '2'
 }
 
-export function getProperties(event: PluginEvent, propertiesToInclude: string): Properties {
+function getNestedProperty(obj: any, path: string): any {
```

Comment:
```suggestion
function getNestedProperty(obj: Record<string, unknown>, path: string): any {
```

i think that's the correct type for obj just from reading

we should specify the return type here too... depends on how strict you want to be but we should "always" try and avoid `any`